src/vs/workbench/contrib/testing/browser/theme.ts

```diff
@@ -190,6 +190,57 @@ export const testStatesToIconColors: { [K in TestResultState]?: string } = {
 	[TestResultState.Skipped]: testingColorIconSkipped,
 };
 
+export const testingRetiredColorIconErrored = registerColor('testing.iconErrored.retired', {
+	dark: transparent('#f14c4c', 0.7),
+	light: transparent('#f14c4c', 0.7),
+	hcDark: transparent('#f14c4c', 0.7),
+	hcLight: transparent('#B5200D', 0.7)
+}, localize('testing.iconErrored.retired', "Retired color for the 'Errored' icon in the test explorer."));
+
+export const testingRetiredColorIconFailed = registerColor('testing.iconFailed.retired', {
+	dark: transparent('#f14c4c', 0.7),
```

Comment:
This is looking better, but for these instead of the color strings you can actually put the original `testingColorIconFailed` variable as the color. This will make those colors automatically derive from that one which makes theming easier

Reply:
Done. The derived color feature is cool!